src/printer.js

```diff
@@ -4256,7 +4262,19 @@ function printAstToDoc(ast, options) {
 
   const doc = printGenerically(FastPath.from(ast));
   docUtils.propagateBreaks(doc);
```

Comment:
Oh, I hadn't considered that (I didn't really think about what `propagateBreaks` does). Fixed in https://github.com/prettier/prettier/pull/1609/commits/070b941c5dd068941e2b3eb6208e528d84095383